src/datasources/tag/TagDataSource.ts

```diff
@@ -69,12 +95,12 @@ export class TagDataSource extends DataSourceBase<TagQuery> {
 
     const response = await this.post<TagsWithValues>(this.tagUrl + '/query-tags-with-values', {
       filter,
-      take: 1,
+      take: 10,
```

Comment:
I think 10 is a reasonable max number of tags to visualize at one time, but I would like PO feedback on this.

@prestwick @joshuaprewitt

Reply:
I would recommend 16 or 32 so that a user could all of the CPUs for a single system or all of the analog inputs for a DAQ board.

Reply:
As "/nitaghistorian/v2/tags/query-history" endpoint can return 10 tags at one query, difference between 16 and 32 will be 2 aditional queries, I think it is not critical. I will set it to 32

Reply:
This is jogging my memory - I had thought the 10 item limitation was only on the frontend in SystemLink, but it looks like the team put additional protection in place on the backend as well. This limit was put in place to mitigate the current performance limitations of the tag service at scale. Hopefully in the future we can raise this limit...

How much extra work is it to add multiple queries here? I'm good starting with 10 in the first pass, but what's the expected delta in complexity?

Reply:
@TJ-G  function to fetch historical data in chunks of 10 tags is ready